teammapper-backend/src/filters/global-exception.filter.ts

```diff
@@ -0,0 +1,62 @@
+import { ExceptionFilter, Catch, ArgumentsHost, Logger } from '@nestjs/common';
+
+// This is for any unhandled gateway and "internal" NestJS related errors - like if the gateway can't reach clients or things like that.
+// It will try to always keep clients and their websockets alive and gracefully send errors over the wire, without revealing internal error reasons.
+@Catch()
+export class GlobalExceptionFilter implements ExceptionFilter {
+  private readonly logger = new Logger(GlobalExceptionFilter.name);
+
+  catch(exception: any, host: ArgumentsHost) {
```

Comment:
maybe you can find a better type here. If you can't avoid any, use unknown and cast later